kura/org.eclipse.kura.driver.ble.sensortag.provider/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,39 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: BLE SensorTag Driver
+Bundle-SymbolicName: org.eclipse.kura.driver.ble.sensortag.provider;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Bundle-Vendor: Eclipse Kura
+Bundle-License: Eclipse Public License v1.0
+Bundle-Category: Asset-Driver Management
+Bundle-ActivationPolicy: lazy
+Import-Package: javax.security.cert,
+ javax.xml,
+ javax.xml.stream,
+ org.eclipse.kura;version="[1.2,2.0)",
```

Comment:
Since we are leveraging the new API, isn't it worth starting the range from 1.3.0?

Reply:
👍